src/zodbupdate/serialize.py

```diff
@@ -298,18 +293,17 @@ def rename(self, input_file):
         return None otherwise.
         """
         self.__changed = False
-        self.__skipped = False
 
         unpickler = self.__unpickler(input_file)
         class_meta = unpickler.load()
```

Comment:
(_Ok, github still eat one of my comment and duplicate all the other ones_). A record is composed of two pickles. We should check for the skipped flag after reading the first one, which contains the class meta information, but not after the second one that contains the data dict. So basically at this point.

Reply:
Okay I changed it to work this way.